README.md

```diff
@@ -1 +1,19 @@
-# phase-0-gps1
\ No newline at end of file
+# phase-0-gps1    7  git push origin pair_2_2
```

Comment:
looks like an old git command from your previous project snuck in here. No big deal tho!